client/quota.go

```diff
@@ -83,3 +83,23 @@ func (client *Client) GetQuotaGroup(groupName string) (*QuotaGroupResult, error)
 	}
 	return res, nil
 }
+
+func (client *Client) RemoveQuotaGroup(groupName string) error {
+	if groupName == "" {
+		return xerrors.Errorf("cannot remove quota group without a name")
+	}
+	data := &postQuotaData{
+		Action:    "remove",
+		GroupName: groupName,
+	}
+
+	var body bytes.Buffer
+	if err := json.NewEncoder(&body).Encode(data); err != nil {
+		return err
+	}
+	if _, err := client.doSync("POST", "/v2/quotas", nil, nil, &body, nil); err != nil {
+		fmt := "cannot remove quota group: %w"
+		return xerrors.Errorf(fmt, err)
```

Comment:
we need to be a bit careful not create together with server errors things that read like:

"cannot remove quota group: cannot remove ..."

we are not very consistent about rewrapping error in Client vs not

Reply:
Fair point, it seems this may be the case (i'm not decorating it in the api, but Ian's backend will report return "cannot remove non-existent quota group.." and "cannot remove quota group with sub-groups...".

I'll revisit this line in a final followup with a spread test.